src/stories/components/Steps.stories.mdx

```diff
@@ -0,0 +1,23 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Steps/Docs" />
+
+# Steps
+
+**Show collaborators what to do next.**
+
+## Overview
+
+The _steps_ component is a helpful piece of UI to let collaborators know what
+completion.
+
+It renders Metecho-flavored vertical SLDS Path with toned down colors as to not
```

Comment:
@jgerigmeyer i don't know how accurate this is; got this from one of the original trello cards for this story, but now I can't seem to recall which one and wanted to link to the actual component in SLDS.